airflow/executors/kubernetes_executor.py

```diff
@@ -347,7 +356,12 @@ def run_next(self, next_job: KubernetesJobType) -> None:
         )
         # Reconcile the pod generated by the Operator and the Pod
         # generated by the .cfg file
-        self.log.info("Creating kubernetes pod for job is %s, with pod name %s", key, pod.metadata.name)
+        self.log.info(
+            "Creating kubernetes pod for job is %s, with pod name %s, annotations: %s",
+            key,
+            pod.metadata.name,
+            annotations_to_str(pod.metadata.annotations),
+        )
```

Comment:
Don't need it here, this already has the key which contains basically the same info:

```
Creating kubernetes pod for job is TaskInstanceKey(dag_id='simple', task_id='world', run_id='manual__2023-03-11T18:59:19.375564+00:00', try_number=1, map_index=-1), with pod name simple-world-58639fac88a74c38a2499d288b40e874
```

Reply:
Thanks for the feedback. Added these in case the `TaskInstanceKey` changes someday, for future proof.
Anyways, removed this